tola/util.py

```diff
@@ -116,3 +119,23 @@ def in_groups(u):
             raise PermissionDenied
         return False
     return user_passes_test(in_groups)
+
+
+def register_in_track(data, tolauser):
```

Comment:
I would avoid to create this function in `util.py`, and would create a new file for this - e.g.: `registration.py`, `registration_bl.py` (bl = Business Logic), `services.py`, or any other better name that you come up with. The reason is that we are going to move to a decoupled package from the CE, and we don't want to use any of the util function. Also, if for easiness we decide to move that function, we just need to copy the file.

Reply:
Moving to registration? Okay... it makes sense but we should also move the RegisterView and other functions related to the registration.